src/main/kotlin/com/kotlin/boardproject/global/handler/OAuth2AuthenticationSuccessHandler.kt

```diff
@@ -1,39 +1,40 @@
-package com.kotlin.boardproject.common.handler
-
-import com.kotlin.boardproject.auth.AuthToken
-import com.kotlin.boardproject.auth.OAuth2UserInfoFactory
-import com.kotlin.boardproject.auth.ProviderType
-import com.kotlin.boardproject.common.config.properties.AppProperties
-import com.kotlin.boardproject.common.enums.ErrorCode
-import com.kotlin.boardproject.common.exception.UnAuthorizedException
-import com.kotlin.boardproject.common.util.addCookie
-import com.kotlin.boardproject.common.util.deleteCookie
-import com.kotlin.boardproject.common.util.log
-import com.kotlin.boardproject.model.User
-import com.kotlin.boardproject.repository.UserRepository
-import com.kotlin.boardproject.repository.common.OAuth2AuthorizationRequestBasedOnCookieRepository
-import com.kotlin.boardproject.repository.common.REDIRECT_URI_PARAM_COOKIE_NAME
-import com.kotlin.boardproject.repository.common.REFRESH_TOKEN
-import com.kotlin.boardproject.repository.common.RedisRepository
+package com.kotlin.boardproject.global.handler
+
+import com.kotlin.boardproject.domain.user.domain.User
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end